### ISSUE TITLE
Explore whether batch size interacts with anything other than effective learning rate

### ISSUE THREAD

**Issue by danbraunai** [goodfire-ai/spd#9]
**Migrated from:** goodfire-ai/spd-gf#55
**Original author:** @leesharkey

---

We have a vague suspicion that when we change the batch size, we are changing more than just the effective learning rate. We are not sure about this though - maybe it is actually just changing the effective learning rate and that's it. 

However, it feels worth exploring this in more detail, even preliminarily, to see if it's interacting with other things. 

I can imagine an exploration looking something like this: 

(**SPOILERS**: you might wish to come up with a plan yourself rather than get prompted with my very low confidence idea for this, which is not very good)

- train SPD on TMS 40-10. Sweep over batch size. Find the range of batch sizes that work/fail. In the region where it fails, fix the batch size to some failing value and sweep over other configs. If any work, then think about why those configs were necessary to change at this batch size.

**Comment by lgngrvs** [comment]
I'll give this a shot, if no one else is working on it

**Comment by danbraunai** [comment]
@lgngrvs want to post your updates on this and close the issue?
